backend/test/look-up-payload-cids.test.js

```diff
@@ -168,9 +171,10 @@ describe('deal-observer-backend piece indexer payload retrieval', () => {
     await lookUpPayloadCids(fetchMinerId, getDealPayloadCid, pgPool, 10000, now)
     assert.strictEqual(payloadsCalled, 1)
     // This is the second attempt that failed to fetch the payload CID so the deal should be marked as unretrievable
-    deal.payload_retrievability_state = PayloadRetrievabilityState.TerminallyUnretrievable
-    deal.last_payload_retrieval_attempt = new Date(now)
-    assert.deepStrictEqual((await loadDeals(pgPool, 'SELECT * FROM active_deals'))[0], deal)
+    const expectedDealDbEntry = { id: 1, ...deal }
+    expectedDealDbEntry.payload_retrievability_state = PayloadRetrievabilityState.TerminallyUnretrievable
+    expectedDealDbEntry.last_payload_retrieval_attempt = new Date(now)
```

Comment:
Nitpick, feel free to ignore.

This can be written as follows:

```js
const expectedDealDbEntry = { 
  id: 1, 
  ...deal,
  payload_retrievability_state: PayloadRetrievabilityState.TerminallyUnretrievable,
  last_payload_retrieval_attempt: new Date(now)
}
```